pallets/proposals/src/tests.rs

```diff
@@ -2395,6 +2395,98 @@ fn test_finalise_milestone_is_ok_on_threshold_vote() {
+    let bob = get_account_id_from_seed::<sr25519::Public>("Bob");
+    let updated_project_name = b"Farmer's Project Sudan2".to_vec().try_into().expect("Invalid input");
+    let expected_project_name_in_event = b"Farmer's Project Sudan2".to_vec().try_into().expect("Invalid input");
+    let updated_project_logo = b"Some logo".to_vec().try_into().expect("Invalid input");
+    let updated_project_description = b"Raise funds for Farmer's project phase 2".to_vec().try_into().expect("Invalid input");
+    let updated_project_website = b"www.ab.com".to_vec().try_into().expect("Invalid input");
+    let additional_amount = 100000000u64;
+    let updated_required_funds = 2_500_000u64;
+    let mut proposed_milestones: Vec<ProposedMilestone> = Vec::new();
+    let milestone1: ProposedMilestone = ProposedMilestone {
+        name: b"milestone 1"
+            .to_vec()
+            .try_into()
+            .expect("input should be of decent length"),
+        percentage_to_unlock: 20,
+    };
+    let milestone2: ProposedMilestone = ProposedMilestone {
+        name: b"milestone 2"
+            .to_vec()
+            .try_into()
+            .expect("input should be of decent length"),
+        percentage_to_unlock: 30,
+    };
+
+    let milestone3: ProposedMilestone = ProposedMilestone {
+        name: b"milestone 3"
+            .to_vec()
+            .try_into()
+            .expect("input should be of decent length"),
+        percentage_to_unlock: 50,
+    };
+    proposed_milestones.push(milestone1);
+    proposed_milestones.push(milestone2);
+    proposed_milestones.push(milestone3);
+
+
+    let mut updated_proposed_milestones: Vec<ProposedMilestone> = Vec::new();
+    let updated_milestone1: ProposedMilestone = ProposedMilestone {
+        name: b"milestone 1"
+            .to_vec()
+            .try_into()
+            .expect("input should be of decent length"),
+        percentage_to_unlock: 70,
+    };
+    let updated_milestone2: ProposedMilestone = ProposedMilestone {
+        name: b"milestone 2"
+            .to_vec()
+            .try_into()
+            .expect("input should be of decent length"),
+        percentage_to_unlock: 30,
+    };
+
+    updated_proposed_milestones.push(updated_milestone1);
+    updated_proposed_milestones.push(updated_milestone2);
+
+
+
+
+    build_test_externality().execute_with(|| {
+        deposit_initial_balance(&alice, &bob, additional_amount);
+        create_project_multiple_milestones(alice, proposed_milestones);
+
+        let project_key = 0;
+
+
+        assert_ok!(Proposals::update_project(
+            Origin::signed(alice), 
+            project_key,
+            updated_project_name,
+            updated_project_logo,
+            updated_project_description,
+            updated_project_website,
+            updated_proposed_milestones.try_into().expect("Invalid proposed milestones"), 
+            updated_required_funds,
+            CurrencyId::Native,
+        ));
+
```

Comment:
I see only an assertion on expected_project_name_in_event. Every property that should be changed by this extrinsic need affirmation.

Reply:
True. That's definitely an input we can take to build additional tests associated to the update_project extrinsic.